src/Pages/Home.js

```diff
@@ -22,34 +25,34 @@ const Home = () => {
     fetchArticles();
   }, [setArticles]);
 
+  const handleClick = (event) => {
+    const findArticle = articles.find(
+      (article) => article.article_id === event.article_id
+    );
+    const articleId = findArticle.article_id;
+
+    if (findArticle) {
+      navigate(`/article/${articleId}`, { state: {article: findArticle}});
+    }
+
+  };
```

Comment:
You shouldn't need to be doing much of this logic here - you are already passing the article into the function